erica_app/tests/elster_xml/sample_with_auth.xml

```diff
@@ -41,6 +41,7 @@
                             <E0100013>X</E0100013>
                         </Belege>
                         <Allg>
+                            <E0100008>017100000001</E0100008>
```

Comment:
FYI: You didn't have to add this to the sample XML. It does no harm, but we usually didn't do this when adding new fields. :)

Reply:
Okay thank you :) The field optional so maybe it makes sense to take it out here again.